docs/quickstart.rst

```diff
@@ -3,7 +3,8 @@ Quickstart
 
 To build a wheel for your setuptools based project::
```

Comment:
This should work with any build back-end, not just setuptools. Maybe that should be reflected in the text?

Reply:
Good idea, updated!